app/mailers/notifier.rb

```diff
@@ -15,6 +15,13 @@ def intro(user, mission: nil)
          subject: t("notifier.welcome", site: Settings.site_name))
   end
 
+  def sms_token_change_alert(mission)
+    @mission = mission
```

Comment:
Can we have a guard clause here requiring mission be non-nil since the recipients can't be determined otherwise.